src/outfit.ts

```diff
@@ -85,11 +87,8 @@ export function freeFightOutfit(requirements: Requirement[] = []): void {
   );
 
   const finalRequirement = new Requirement(parameters, {
-    forceEquip: forceEquip,
-    preventEquip: [
-      ...preventEquip,
-      ...$items`broken champagne bottle, Spooky Putty snake, Spooky Putty mitre, Spooky Putty leotard, Spooky Putty ball, papier-mitre, smoke ball`,
-    ],
+    forceEquip,
+    ...(preventEquip.length > 0 ? { preventEquip } : {}),
```

Comment:
We use preventEquip for the bjorn/crown much of the time--wouldn't this perpetually override our default settings?

Reply:
oh okay, I'll have to scrub the code for that